src/test/kotlin/org/rust/RustProjectDescriptors.kt

```diff
@@ -435,6 +440,16 @@ private class WithStdlibLikeDependencyRustProjectDescriptor : RustProjectDescrip
  */
 object WithStdlibAndStdlibLikeDependencyRustProjectDescriptor : WithRustup(WithStdlibLikeDependencyRustProjectDescriptor())
 
+/**
+ * The same as [org.rust.DefaultDescriptor] but have [PackageOrigin.STDLIB] instead of [PackageOrigin.WORKSPACE]
```

Comment:
```suggestion
 * The same as [org.rust.DefaultDescriptor] but it provides test package with [PackageOrigin.STDLIB] origin instead of [PackageOrigin.WORKSPACE]
```